tap-snapshots/test/lib/commands/pack.js.test.cjs

```diff
@@ -7,11 +7,10 @@
 'use strict'
 exports[`test/lib/commands/pack.js TAP dry run > logs pack contents 1`] = `
 Array [
-  undefined,
   "package: test-package@1.0.0",
-  undefined,
+  "=== Tarball Contents ===",
```

Comment:
This is because `log-tar` calls `log.notice` with no preceding verb.  Your `args.length === 4` is catching this now?

Reply:
The `args.length === 4` is only for rewriting ERESOLVE explanations. This is likely another culprit of catching logs from `process.stderr.write` instead of `process.on('log')`.

But all of these changes are good to call out since they will require investigation to be sure.

Reply:
Ohhh ok, that `if` statement is way less worrying now and probably can be something we revisit later.